backend/internal/infrastructure/mailservice/mail_service.go

```diff
@@ -21,6 +21,7 @@ type MailService interface {
 	InvoiceMailContent(invoiceTotal float64, currency string, invoiceID int) (string, string)
 	SystemAnnouncementMailBody(body string) string
 	NotifyAdminsMailContent(recordsNumber int) (string, string)
+	SendBulkSystemMails(receivers []string, body string, subject string, attachments ...Attachment) int
```

Comment:
I would let the service (adminService) use the SendMailFromSystem function to implement the bulksending instead of adding extra function to the interface, but that's just me

> note the interface needs to get split e.g formatter and mailer imo